src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -12,22 +12,26 @@ import MessageList from "./MessageList.vue";
 import ConfirmDialog from "../ConfirmDialog.vue";
 import PaginationStrip from "../../components/PaginationStrip.vue";
 import moment from "moment";
+import type { Ref } from "vue";
+import { FailedMessageViewWithExtendedUIProperties } from "@/resources/FailedMessageView";
+import Configuration from "@/resources/Configuration";
+import { TYPE } from "vue-toastification";
 
 let pollingFaster = false;
-let refreshInterval = undefined;
+let refreshInterval: ReturnType<typeof setInterval>;
 const perPage = 50;
-const configuration = ref([]);
+const configuration: Ref<Configuration | null> = ref(null);
```

Comment:
this works without importing Ref
```suggestion
const configuration = ref<Configuration | null>(null);
```